packages/turbo-ignore/package.json

```diff
@@ -25,16 +25,18 @@
     "check-types": "tsc --noEmit",
     "lint:prettier": "prettier -c . --cache --ignore-path=../../.prettierignore"
   },
+  "dependencies": {
+    "commander": "^11.0.0",
+    "@turbo/telemetry": "workspace:*"
```

Comment:
Hmm, yeah was wondering if the dependencies would matter since it's a published bundled, but I didn't go all the way to test it out. I think i'll just close this out in that case. It doesn't make sense to just add `commander` to deps